packages/angular_devkit/build_angular/src/babel/webpack-loader.ts

```diff
@@ -109,7 +112,7 @@ export default custom<AngularCustomOptions>(() => {
           [
             require('./presets/application').default,
             {
-              angularLinker: customOptions.shouldLink,
+              angularLinker: customOptions.angularLinker,
```

Comment:
nit: Since the property names all line up now, could probably be cleaned up with `...customOptions`.